### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by adamalston** [adamalston/v2#80]
All accessibility violations should be addressed to improve site accessibility.

[accessibility-report-dark.xlsx](https://github.com/adamalston/v2/files/13624588/accessibility-report-dark.xlsx)

![dark-accessibility-checker](https://github.com/adamalston/v2/assets/18297826/271a9437-4f95-479b-a5de-a293c59b99c7)

[accessibility-report-light.xlsx](https://github.com/adamalston/v2/files/13624589/accessibility-report-light.xlsx)

![light-accessibility-checker](https://github.com/adamalston/v2/assets/18297826/c27fd3be-8a58-460d-8242-10c31b9094a3)
